src/mscorlib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -61,6 +61,9 @@ private struct Entry
         private const string KeyValuePairsName = "KeyValuePairs"; // Do not rename (binary serialization)
         private const string ComparerName = "Comparer"; // Do not rename (binary serialization)
 
+        private static ConditionalWeakTable<object, SerializationInfo> s_serializationInfoTable;
```

Comment:
This will include the static in every instantation of Dictionary. This needs to stay in a static class outside the dictionary.

Reply:
E.g. `HashHelpers.SerializationInfoTable.cs`